sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorAspNetCoreEventSource.cs

```diff
@@ -53,5 +53,14 @@ public void ConfigureFailed(Exception ex)
 
         [Event(1, Message = "Failed to configure AzureMonitorOptions using the connection string from environment variables due to an exception: {0}", Level = EventLevel.Error)]
         public void ConfigureFailed(string exceptionMessage) => WriteEvent(1, exceptionMessage);
+
+        [Event(2, Message = "Package reference for {0} found. Backing off from its vendor instrumentation.", Level = EventLevel.Warning)]
```

Comment:
This statement is a little confusing to me. How about something like this?

```suggestion
        [Event(2, Message = "Package reference for {0} found. Backing off from default included instrumentation. Action Required: You must manually add and configure this instrumentation.", Level = EventLevel.Warning)]
```